src/rendering/SceneManager.cc

```diff
@@ -2235,6 +2153,105 @@ AnimationUpdateData SceneManagerPrivate::ActorTrajectoryAt(
   return animData;
 }
 
+/////////////////////////////////////////////////
+std::vector<common::TrajectoryInfo>
+SceneManager::LoadTrajectories(const sdf::Actor &_actor,
+  std::unordered_map<std::string, unsigned int> &_mapAnimNameId,
+  Entity &_id)
+{
+  rendering::MeshDescriptor descriptor;
+  descriptor.meshName = asFullPath(_actor.SkinFilename(), _actor.FilePath());
+  common::MeshManager *meshManager = common::MeshManager::Instance();
+  descriptor.mesh = meshManager->Load(descriptor.meshName);
+  common::SkeletonPtr meshSkel = descriptor.mesh->MeshSkeleton();
```

Comment:
The logic to get a skeleton from a descriptor is already done outside the function at line 963. Can't we just pass the skeleton got there to the function, so we don't need to retrieve it again?

Reply:
Done.